app/uninstall.go

```diff
@@ -94,6 +94,7 @@ func uninstall(c *cli.Context) error {
 	persistentVolumeClaimInformer := kubeInformerFactory.Core().V1().PersistentVolumeClaims()
 	kubeNodeInformer := kubeInformerFactory.Core().V1().Nodes()
 	priorityClassInformer := kubeInformerFactory.Scheduling().V1().PriorityClasses()
+	csiDriverInformer := kubeInformerFactory.Storage().V1beta1().CSIDrivers()
```

Comment:
CSI driver informer? the commit message only mentioned the deployments.

Also, why the sync period doesn't help?

Reply:
In this case, all related CRs, deployements and daemonsets are gone hence there is no event for the uninstaller controller, the 30s-force-sync doesn't work.